src/components/LocationDetails.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReactTooltip from "react-tooltip";
+import Collapsible from "react-collapsible";
+import propsMapping from "./propsMapping";
+
+function LocationDetails(props) {
+  const featureProps = props.feature.properties;
+
+  const renderMetric = (metric) => {
+    let value = featureProps[metric.id];
+    switch (metric.format) {
+      case 'number':
+        if (value > 100) {
+          value = Math.floor(value)
+          value = value.toLocaleString('en-US');
+        }
+        else {
+          value = Math.floor(value * 10000) / 10000;
+          value = value.toLocaleString('en-US', { minimumFractionDigits: 4 });
+        }
+        break;
+      case 'currency':
+        value = value.toLocaleString(undefined, { style: "currency", currency: "AUS" });
+        break;
+      case 'percent':
+        value = `${Math.floor(value)}%`;
+        break;
+      default:
+        break;
+    }
+
+    return (
+      <div key={metric.id}>
+        <h2 data-tip data-for={metric.id}>{metric.label}</h2>
+        {metric.desc && (
+          <ReactTooltip id={metric.id}>
+            <strong>{metric.label}</strong>
+            <div style={{maxWidth: 400}}>
+              {metric.desc}
+            </div>
+          </ReactTooltip>
+        )}
+        <p>{value}</p>
+      </div>
+    )
+  }
+
+  return (
+    <div
+      style={{ overflowY: "auto" }}
+      className={`sidebar-content`}
+    >
+      {propsMapping.map((section) => (
```

Comment:
Could you make LocationDetails render `{props.children}` here, so it's possible to pass other collapsible sections in to prepend this overflowing div? The "Locations to Compare" work Ethan's doing in #104 will need to be passed along into there, as well as other components in the future. So instead of what Ethan's currently got in sidebar.js, https://github.com/CxSci/SA-dashboard/blob/203dabfd575d667aaed0dfe87f32bb9744c81ff7/src/components/sidebar.js#L67-L69, he'd do:
```js
<LocationDetails feature={this.props.selectedFeature}>
  <LocationCompare />
</LocationDetails>
```